docker/accessdata/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/AccessDataOps/FTK-API-SDK.git@a3d9281586f25fdbef9cfab7e0f0082321b8afa6
```

Comment:
Why do you want to use this particular commit rather than the branch name?

Reply:
previously i had branch name only, but when changes were made in FTK-API-SDK, docker build is getting failed because there would be no change in dockerfiles/docker/accessdata folder. So changed to commit hash from branch name. (once docker image is built, sdk will be cached in the image as a layer). for each change in the ftk-api-sdk, i will update the corresponding commit hash value in the requirements.txt file, so that build will not fail and will get new image tag for each.(adding @kgal-pan for reference, we had discussion)

Reply:
@YuvHayun  - For reference, see [the PR](https://github.com/demisto/dockerfiles/pull/8460/files) where the branch name was used. As you can see, the branch name was used there but since the branch was updated, there is no diff in the `Dockerfile`. Is there a way to update the Docker image when there are no diffs in the `Dockerfile`? 

I believe this is the reason @sriram7797 used a commit hash instead of branch name.

Reply:
@kgal-pan Yes that's the reason i used commit. 

basically each command in the Dockerfile will be cached as the layer in the machine on which image is built. when we build docker image again without changes in the Dockerfile, it will take the cached layer from machine (command wont be executed to build the docker image). we need to build docker image with **--no-cache** option. i am just not sure this could also be problem. (came across this scenario when building test build locally using ./build_docker.sh)